test/test_torch.py

```diff
@@ -13852,10 +13852,11 @@ def test_float_scalar_pow_float_tensor(self, device):
     @onlyCUDA
     @unittest.skipIf(not TEST_NUMPY, 'Numpy not found')
     def test_cuda_tensor_pow_scalar_tensor(self, device):
-        cuda_tensor = torch.randn((3, 3), device=device)
-        scalar_tensors = [torch.tensor(5), torch.tensor(-3), torch.tensor(1)]
-        for exp in scalar_tensors:
-            self._test_pow(cuda_tensor, exp)
+        cuda_tensors = [torch.randn((3, 3), device=device), torch.tensor(3.0, device=device)]
+        scalar_tensors = [torch.tensor(5.0), torch.tensor(-3), torch.tensor(1)]
+        for base in cuda_tensors:
```

Comment:
nit: 
```
for base, exp in product(cuda_tensors, scalar_tensor):
```
You should also add tests that assert errors (self.assertRaisesRegexp) when you are trying to do cpu base and cuda 0-d exponent.